src/modm/ui/display/graphic_display.hpp

```diff
@@ -34,75 +35,19 @@ namespace modm
 		/// @ingroup modm_ui_display
 		typedef Vector<int16_t, 2> Point;
 
-		// RGB16 (565) Format
+		// Backwards compatibility
 		/// @ingroup modm_ui_display
-		class Color
-		{
-		public:
-			static modm_always_inline Color white()   { return Color(0xffff); };
-			static modm_always_inline Color yellow()  { return Color(0xFFE0); };
-			static modm_always_inline Color magenta() { return Color(0xF81F); };
-			static modm_always_inline Color red()     { return Color(0xF800); };
-			static modm_always_inline Color orange()  { return Color(0xFD20); };
-			static modm_always_inline Color sliver()  { return Color(0xC618); };
-			static modm_always_inline Color gray()    { return Color(0x8410); };
-			static modm_always_inline Color maroon()  { return Color(0x8000); };
-			static modm_always_inline Color lime()    { return Color(0x07E0); };
-			static modm_always_inline Color green()   { return Color(0x0400); };
-			static modm_always_inline Color blue()    { return Color(0x001F); };
-			static modm_always_inline Color navy()    { return Color(0x0010); };
-			static modm_always_inline Color black()   { return Color(0x0000); };
-			static modm_always_inline Color signalViolet()   { return Color(0x8010); }; //0x84D0
-			static modm_always_inline Color emeraldGreen()   { return Color(0x5DCC); };
-
-			/**
-			 * @param	red
-			 * 		Range [0..255]
-			 * @param	green
-			 * 		Range [0..255]
-			 * @param	blue
-			 * 		Range [0..255]
-			 */
-			Color(uint8_t red, uint8_t green, uint8_t blue) :
-				color(((static_cast<uint16_t>(red >> 3) << 11) |
-						(static_cast<uint16_t>(green >> 2) << 5) |
-						static_cast<uint16_t>(blue >> 3)))
-			{
-			}
-
-			Color(uint16_t color) :
-				color(color)
-			{
-			}
-
-			Color() : color(0)
-			{
-			}
-
-			inline uint16_t
-			getValue() const
-			{
-				return color;
-			}
-
-			bool
-			operator == (const Color& other) const {
-				return (color == other.color);
-			}
-
-		private:
-			uint16_t color;
-		};
+		using Color = color::RgbHtml;
```

Comment:
Ah, I see… but using constants instead of functions will break backwards compatibility anyways. I'm ok with that.
The fix for users will be a relatively simple regex find/replace, I think that's fair.

Reply:
Yes.. this was the reason for the RgbHtml class. Much better without and if you're OK with breaking (more) stuff, i'm in!